python/src/aiconfig/schema.py

```diff
@@ -136,9 +136,11 @@ class Error(BaseModel):
 
 
 class ModelMetadata(BaseModel):
-    # The ID of the model to use.
+    # The ID of the inference provider aka Model Parser to use.
```

Comment:
How would these parser be registered? Would it be `registerModelParser(new Parser(model))` for some list of models, instead of `registerModelParser(new Parser(), [model list])`? This is one thing we should address with this update

Reply:
for this method which would essentially unlink the model from model parser, it would be

`registerModelParser(new Parser())`

whereas the other notation would like this:

`registerModelParser(new Parser(), [modelParserAlias list])`

Reply:
>This is one thing we should address with this update

+1 from my side, I think the problem boils down to how do we id these and how does that setup look like